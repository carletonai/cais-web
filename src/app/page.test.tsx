import React from "react";
import { render } from "@testing-library/react";
import HomePage from "./page";
test("renders Home page correctly", () => {
  const { asFragment } = render(<HomePage />);
  expect(asFragment()).toMatchSnapshot();
});
