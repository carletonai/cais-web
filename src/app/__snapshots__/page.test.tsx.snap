// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home page correctly 1`] = `
<DocumentFragment>
  <div
    class="min-h-screen p-8"
  >
    <h1
      class="text-4xl font-bold mb-6"
    >
      CAIS
    </h1>
    <p
      class="text-lg"
    >
      Welcome to Carleton's Artificial Intelligence Society.
    </p>
  </div>
</DocumentFragment>
`;
